client/package.json

```diff
@@ -9,7 +9,8 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-scripts": "5.0.0",
+    "react-router-dom": "^6.3.0",
```

Comment:
did the you change this or did it change automatically when you updated the packages? jw if i need this to change on my end and update my packages.

Reply:
Yes you need to install npm install react-router-dom@6 to make this run!